libs/coin-framework/types/compressjs/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '@ledgerhq/compressjs' {
```

Comment:
note to ourself that we may need to extract the Bzip2 out of compressjs to type it properly in future & not maintain this fork. cc @valpinkman @jelbaz-ledger too